### ISSUE TITLE
Not able to input second diget for day if mount is 01 for format mm/dd/yyyy. for v4.0.0-100

### ISSUE THREAD

**Issue by vtarasenko** [RobinHerbots/Inputmask#1808]
Hi,

using the following config:
{
                    alias: "datetime",
                    inputFormat: 'mm/dd/yyyy',
                    placeholder: 'mm/dd/yyyy',
                    showMaskOnFocus: false,
                    clearMaskOnLostFocus: false,
                    showMaskOnHover: false,
                    min: minDate.format("MM/DD/YYYY"),
                    max: maxDate.format("MM/DD/YYYY")
}

It will stop on position 01/1d/yyyy

**Comment by RobinHerbots** [comment]
@vtarasenko , 

What is the value of mindate and maxdate?

**Comment by vtarasenko** [comment]
var minDate = moment().subtract(80, 'years');
var maxDate = moment().add(20, 'years');

**Comment by RobinHerbots** [comment]
@vtarasenko , 

Can you try without the min and max.  Probably there is still a bug in it

**Comment by vtarasenko** [comment]
it works fine without min/max.

**Comment by RobinHerbots** [comment]
@vtarasenko , 

Ok thx for the info, I will push a fix soon

**Comment by vtarasenko** [comment]
Thanks

**Comment by RobinHerbots** [comment]
@vtarasenko , 

Can you send a example of the output of minDate.format("MM/DD/YYYY"), and maxdate

**Comment by vtarasenko** [comment]
min: 02/12/1938 
max: 02/12/2038

**Comment by RobinHerbots** [comment]
@vtarasenko , 

Can you retry with beta 14

**Comment by vtarasenko** [comment]
Was tested with:
min: "02/14/1938"
max: "02/14/2038" 

Problem also exists for example:
I can't insert year started with 1 if month and day is less than min month and day, 
It will stop on position 01/01/yyyy, but value 2 for year is allowed 01/01/2yyy
value 1 for year is allowed If I insert 02/14/1yyy or 03/01/1yyy

**Comment by RobinHerbots** [comment]
@vtarasenko , 

and now there is version beta 17 ;-)
Can you have another try.

**Comment by vtarasenko** [comment]
Hi, previous issues were fixed, BUT there is another one now, here are steps to reproduce:
1. enter date 11/05/2018
2. navigate to digits 05, click on backspace 2 times (remove it using the backspace)
3.  now the date/time is broken - you can not enter any date (because the year part shifts left onto the day section)

**Comment by RobinHerbots** [comment]
Yes, hmm what would be a good approach?  Will it feel intuitive when I
don't shift?

Op do 15 feb. 2018 16:43 schreef vtarasenko <notifications@github.com>:
